.jenkins/.pipeline/lib/config.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const options= require('pipeline-cli').Util.parseArguments()
+const changeId = options.pr //aka pull-request
+const version = '1.0.0'
+const name = 'jenkins-realm-o-matic'
+
+const phases = {
+  build: {namespace:'devhub-tools'    , name: `${name}`, phase: 'build'  , changeId:changeId, suffix: `-build-${changeId}`  , instance: `${name}-build-${changeId}`  , version:`${version}-${changeId}`, tag:`build-${version}-${changeId}`},
+  dev: {namespace:'devhub-tools'    , name: `${name}`, phase: 'dev'  , changeId:changeId, suffix: `-dev-${changeId}`  , instance: `${name}-dev-${changeId}`  , version:`${version}-${changeId}`, tag:`dev-${version}-${changeId}`},
+  test: {namespace:'devhub-tools'    , name: `${name}`, phase: 'test'  , changeId:changeId, suffix: `-test`  , instance: `${name}-test`  , version:`${version}`, tag:`test-${version}`},
+  prod: {namespace:'devhub-tools'    , name: `${name}`, phase: 'prod'  , changeId:changeId, suffix: `-prod`  , instance: `${name}-prod`  , version:`${version}`, tag:`prod-${version}`},
+};
+
+// This callback forces the node process to exit as failure.
+process.on('unhandledRejection', (reason) => {
+  console.log(reason);
+  process.exit(1);
+});
+
+module.exports = exports = {phases, options};
```

Comment:
i'm not too sure about that this line, why does `exports` need to be assigned that object?

couldn't this just be `module.exports = { ... }`